test/integration/steps/test_steps.py

```diff
@@ -645,6 +647,32 @@ def check_service_history_specified_times(context: Context, added_or_removed):
     return context
 
 
+@then(parse('the service history is updated with the "{added_or_removed}" standard opening times'))
+def check_service_history_standard_times(context: Context, added_or_removed):
+    openingtimes = context.change_event.standard_opening_times
+    dos_times = get_service_history_standard_opening_times(context.service_id)
+    expected_dates = convert_standard_opening(openingtimes)
+    counter = 0
+    if added_or_removed == "added":
+        for entry in dos_times:
+            currentday = list(entry.keys())[0]
+            for dates in expected_dates:
+                if dates["name"] == currentday:
+                    assert entry[currentday]["data"]["add"][0] == dates["times"], "Dates do not match"
```

Comment:
Please can an assertion be added to check the is no remove from in the data field.
``` assert "remove" not in entry[currentday]["data"]```